src/column_type.rs

```diff
@@ -0,0 +1,89 @@
+use pyo3::prelude::{PyModule, PyModuleMethods};
+use pyo3::{Bound, Py, PyAny, PyResult, Python, pyclass, pymodule};
+
+
+#[pyclass(subclass)]
+pub struct PyNativeType;
+
+#[pyclass(extends=PyNativeType)]
+pub struct Double;
```

Comment:
:question: Is `subclass` transitive for all subclasses or not? Will `Double` be inheritable in Python or not? Which is the desired behaviour?